src/or/entrynodes.c

```diff
@@ -1411,11 +1413,11 @@ sampled_guards_update_from_consensus(guard_selection_t *gs)
     approx_time() - get_guard_confirmed_min_lifetime();
 
   /* Then: remove the ones that have been junk for too long */
+  n_changes +=
   sampled_guards_prune_obsolete_entries(gs,
```

Comment:
Please intend the code here! :)

Reply:
Fixed in 870dab729af6bec6f1d01378a104ba8f25490b74.